add-on/chrome/bootstrap.js

```diff
@@ -485,6 +488,8 @@ var WindowListener = {
           // metadata about the page is available when this event fires.
           gBrowser.addEventListener("DOMTitleChanged", this);
           this._browserSharePaused = false;
+
+          gBrowser.addEventListener("mousemove", this);
```

Comment:
Add a comment about how this is added to the parent `gBrowser` to avoid adding/removing for each individual tab's browsers.